packages/medusa/src/loaders/api.ts

```diff
@@ -1,4 +1,5 @@
-import { Express } from 'express'
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
question: Why this rule has been disabled here?

Reply:
reverted,